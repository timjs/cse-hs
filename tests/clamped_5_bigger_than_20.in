5
f(baz,plus(x(baz(a,mult),plus(f,b)),baz(plus,a)))
minus(f(baz(minus,pow),plus),y(f,f(a(a,f),mult)))
plus(y(y(b,x(a,f(pow,b))),x(mult(x,pow),y)),plus)
b(a(pow(pow(y,x),baz),b(baz,mult)),f(a,b(a,baz)))
baz(baz,mult(pow(f,a),f(plus(mult,mult),y(f,x))))
