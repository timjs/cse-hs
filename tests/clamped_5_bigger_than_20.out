f(baz,plus(x(baz(a,mult),plus(f,b)),baz(plus,6)))
minus(f(baz(minus,pow),plus),y(f,f(a(a,8),mult)))
plus(y(y(b,x(a,f(pow,4))),x(mult(x,8),y)),plus)
b(a(pow(pow(y,x),baz),b(7,mult)),f(a,b(11,7)))
baz(baz,mult(pow(f,a),f(plus(mult,9),y(5,x))))
